src/components/Forum/Item.js

```diff
@@ -1,69 +1,43 @@
-                createdAt: Date.now()}
-            ]
-            ,
-            email: 'email',
-          
-          });
-        //     const voteRef = firebase.database().ref('forum').push.doc(question.uid)
-        //     voteRef.get().then(doc => {
-            
-        //         if (doc.exists) {
-        //             const previousVotes = doc.data().votes;
-        //             const vote = { votedBy: { id: 'unknown', name: 'unknown' } }
-        //             const updatedVotes = [...previousVotes, vote];
-        //             voteRef.update({ votes: updatedVotes })
-        //         }
-            
-        // }
-        //     )
-    }
+    firebase.database().ref('forum/' + question.qid).update({
+      votedBy: [
+        ...question.votedBy,
+        {
+          userid: 'krzychi',
+          createdAt: Date.now()
+        }
+      ],
+      email: 'email',
+    });
+  }
 
-    
-    function handleDelete() {
-   
-        firebase.database().ref('forum').child(question.qid).remove()
-    }
-    return (
-   
-            <div className={styles.mainDiv}>
-
-                <div>
-                    <div className={styles.title}>  {question.title}</div>
-                  <div>{question.description}</div> 
-                </div>
-                
-                <div>
-    LIKES{question.votedBy.length-1}
-    {/* Recent like: {formatDistanceToNow(question.votedBy[question.votedBy.length-1].createdAt)} */}
-                </div>
-                <Link to={`/forum/${question.qid}`}> 
-                   DISCUSS
-                 </Link>
-                 <div onClick={handleVote}>
-                    Łapka w góre
-                   
-</div>
-                 {/* {question.comments.length > 0 ? `${question.comments.length} comments` : "discuss"} */}
-              <span onClick={handleDelete}>DELETE</span>
-                
-            </div>
-       
-    )
-    
+  function handleDelete() {
+    firebase.database().ref('forum').child(question.qid).remove()
+  }
+  return (
+    <div className={styles.mainDiv}>
+      <div className={styles.questionSection}>
+        <div className={styles.title}>  {question.title}</div>
+        <div className={styles.description}>{question.description}</div>
+        <div style={{ display: "flex" }}>
+          <Link to={`/forum/${question.qid}`} style={{ margin: "0", padding: "2px", fontSize: "120%" }}>DISCUSS</Link>
+          <div onClick={handleDelete} style={{ border: "1pxsolid black", margin: "0", padding: "2px", fontSize: "120%" }}>DELETE</div>
+        </div>
+      </div>
+      <div className={styles.likesSection}>
+        <div style={{ margin: "0" }}>{question.votedBy.length - 1}</div>
+        <div style={{ margin: '0', cursor: "pointer", display: "flex", justifyContent: "center", alignItems: "center", paddingTop: "35px", fontSize: "200%" }} onClick={handleVote}>
+          <span>^</span>
+        </div>
+      </div>
+    </div>
+  )
 }
```

Comment:
Looks fine - inline styles and fixed user has to be changed but it will come with next issues.